Tests/Twig/Extension/SonataAdminExtensionTest.php

```diff
@@ -2002,6 +2130,24 @@ public function testGetUrlsafeIdentifier_GivenAdmin_Bar()
         $this->assertSame(1234567, $this->twigExtension->getUrlsafeIdentifier($entity, $this->adminBar));
     }
 
+    /**
+     * Remove this method for next major release only if PHP 5.3 support will be dropped
```

Comment:
After the line, not before. :stuck_out_tongue: 

Also add a blank line between the two to have a clearer phpdoc.

And add a dot at the end, or StyleCI will cry. :wink: